Pipfile

```diff
@@ -8,8 +8,9 @@ black = "*"
 coincurve = "*"
 cryptography = "*"
 eth-ape = "*"
-# TODO eventually change to official release, issue #82, once fix is available
 ape-solidity = ">=0.6.5"
+ape-etherscan = "*"
+ape-polygon = "*"
```

Comment:
Maybe we can consider to close https://github.com/nucypher/nucypher-contracts/issues/96? Although I'd reconsider to relocate as a dev dependency.

Reply:
👍 . There are a few others like `nucypher_core` that are only needed for tests that we can consider moving to dev dependencies.

I guess it depends on what counts as `dev` vs regular dependencies.  `nucypher_core` is clear to me as dev (only for tests), but `ape-etherscan` etc. is needed for deployment scripts...is that dev? I guess so...